src/components/Login.js

```diff
@@ -1,56 +1,59 @@
 import { useState, useEffect, useRef } from "react";
+import axiosInstance from "../api/axios";
+import { Link } from "react-router-dom";
+// import Navbar from "./Navbar";
 
 const Login = () => {
-  const errRef = useRef();
-
-  const [user, setUser ] = useState({username: "", password: ""});
-  const [errMsg, setErrMsg] = useState("");
+  const [user, setUser] = useState({ username: "", password: "" });
+  const [errLog, setErrLog] = useState("");
   const [success, setSuccess] = useState(false);
 
   const handleSubmit = async (e) => {
     e.preventDefault();
-    setUser("");
-    setSuccess(true);
+    try {
+      if (user.username === "" || user.password === "") {
+        setErrLog("Please provide your login details");
+        setSuccess(false);
+      } else if (
+        user.username === user.password ||
+        user.password === "undefined"
+      ) {
+        setErrLog("Please provide valid username or password");
+        setSuccess(false);
+      } else {
+        axiosInstance.post("login", user);
+        setSuccess(true);
+      }
+    } catch (error) {
+      console.log(error);
+    }
   };
 
-  const getUserDetails = (event) => {
-    setUser((current) => ({...current, [event.target.name]: event.target.value,}))
-
-    // to validate user
-    // if(setUser === user.username) {
-    //   errMsg('')
-    // }
+  const getUserDetails = async (e) => {
```

Comment:
`handleChange` or `handleUserDetailsChange` or any name with `handle` in it is a better name.